lib/utils/promote.ts

```diff
@@ -334,13 +334,7 @@ async function createApplication(
 				try {
 					await sdk.models.application.getDirectlyAccessible(appName, {
 						$filter: {
-							$or: [
-								{ slug: { $startswith: `${username!.toLowerCase()}/` } },
-								// TODO: do we still need the following filter? Is it for
-								// old openBalena instances where slugs were equal to the
-								// app name and did not contain the slash character?
-								{ $not: { slug: { $contains: '/' } } },
-							],
+							$or: [{ slug: { $startswith: `${username!.toLowerCase()}/` } }],
```

Comment:
Pine would error with this $or that has only a single item :P
```suggestion
							slug: { $startswith: `${username!.toLowerCase()}/` },
```